docs/G--Model-System/statuses.md

```diff
@@ -0,0 +1,122 @@
+* `RNA` not approved - Payments are NOT allowed. Event Admin must manually toggle STS_ID for it to change. No space reserved. Registration is active.
+* `RPP` pending payment - Payments are allowed and the status will automatically be toggled to `RAP` if payment is made in full by the attendee. No space reserved. Registration is active
+* `RWL` wait list - The registration has been placed on the waiting list (see the wait-list add-on). Payments are allowed. The status will automatically be toggled to RAP if payment is made in full by the attendee. No space reserved. Registration is active.
+* `RAP` approved - The registration has been approved (although the transaction may not yet be paid in full). Further payments are permitted if required. A space IS reserved. Registration is active.
+* `RCN` cancelled - The registration was cancelled. Payments are NOT allowed. NO space reserved. Registration is NOT active
+* `RDC` declined - The registration was the event manager. Payments are NOT allowed. No space reserved. Registration is NOT active
+
+### Transactions
+Transactions are basically orders for one or more registrations. Their status is related to registrations, although they can be changed independently.
+
+* `TFL` failed - Either due to a technical reason (server or computer crash during registration), or some other reason that prevent the collection of any useful contact information from any of the registrants.
+* `TAB` abandoned - Either due to a technical reason (server or computer crash during registration), or due to an abandoned cart after registrant chose not to complete the registration process HOWEVER... an abandoned TXN differs from a failed TXN in that it was able to capture contact information for at least one registrant. When users are sent for payment offsite, their transaction is immediately updated to "TAB", but updated to something else upon their return.
+* `TIN` incomplete - Meaning that monies are still owing
+* `TCM` complete - No monies are owing
+* `TOP` overpaid - This is the same as complete, but site admins actually owe clients the moneys! 
+
+## Soft Deleted/Archived/Trashable Models
+These are models with a special field of type `EE_Trashed_Flag_Field`, which indicates if the item is either soft-deleted/trashed/archived (the terms are used interchangeably.)
+Note that when you're querying these models, trashed items are automatically omitted unless you explicitly include them. See their common parent class `EE_Soft_Delete_Base` for helper methods. 
+
+### Datetimes
+Trashed datetimes don't appear on the frontend, but are still visible by event managers.
+
+### Tickets
+Tickets are automatically archived (trashed) when their price changes. (This is arguably unnecessary since the line items table was introduced.)
```

Comment:
> (This is arguably unnecessary since the line items table was introduced.)

i think there are still places that compute transaction totals directly from ticket prices, so might want to remove this

Reply:
Oh I wasn't aware. Yeah if this is wrong I'll remove.